zebedee-reader/src/main/java/com/github/onsdigital/zebedee/content/page/home/HomePage.java

```diff
@@ -40,6 +41,14 @@ public void setSections(List<HomeSection> sections) {
         this.sections = sections;
     }
 
+    public List<HomeContentItem> getFeaturedContent() {
```

Comment:
Its probably advisable to initialise these fields in the constructor so they are not null.
```
private List<HomeSection> sections;
private List<HomeContentItem> featuredContent;
```
Something like
```
this.sections = new ArrayList<>();
this.featuredContent = new ArrayList():
```